### PR TITLE
Enable crossgen tests

### PR DESCRIPTION
**Pull request by billwert**
This turns on benchmarking for a few binaries to track crossgen.

### CODE REVIEW

**Comment by billwert** [comment]
This PR does some support work that is necessary when the PR from arcade comes in which will enable the tests, so it has to go first. I'd like to get this merged ASAP and then put the PR in arcade which in theory will flow in tomorrow.

File: eng/pipelines/coreclr/templates/run-performance-job.yml [hunk 1]
```diff
@@ -83,7 +83,15 @@ jobs:
       continueOnError: ${{ parameters.continueOnError }}
     - script: $(Python) $(PerformanceDirectory)/scripts/ci_setup.py $(SetupArguments)
       displayName: Run ci setup script
-    # Run perf testing in helix
+    - script: xcopy $(PerformanceDirectory)\scripts $(WorkItemDirectory)\ScenarioCorrelation\scripts\/e && xcopy $(PerformanceDirectory)\src\scenarios\shared $(WorkItemDirectory)\ScenarioCorrelation\shared\/e && xcopy $(PerformanceDirectory)\src\scenarios\staticdeps $(WorkItemDirectory)\ScenarioCorrelation\staticdeps\/e
```
Comment (BruceForstall) [inline eng/pipelines/coreclr/templates/run-performance-job.yml]: Does the `/e` argument need a preceding space?

**Review by BruceForstall — APPROVED** [review]
I'm not familiar with this code, so I can't comment much on it. Hopefully you can test if it works before merging? Or do you need to wait for arcade changes to come in first?

File: eng/pipelines/coreclr/templates/run-performance-job.yml (hunk 1 above)
Comment (billwert) [reply eng/pipelines/coreclr/templates/run-performance-job.yml]: Nope. Works fine. :)

**Comment by billwert** [comment]
> I'm not familiar with this code, so I can't comment much on it. Hopefully you can test if it works before merging? Or do you need to wait for arcade changes to come in first?

Good question. Before I force pushed and cleaned it up I had the change that's going into arcade in this PR. Here's output from the last CI run showing it worked:
```
C:\h\w\A4080960\w\B66A0983\e>py -3 C:\h\w\A4080960\p\performance\src\scenarios\crossgen\test.py crossgen --test-name Microsoft.CodeAnalysis.CSharp.dll --core-root C:\h\w\A4080960\p\Core_Root 
[2020/02/26 18:22:21][INFO] ----------------------------------------------
[2020/02/26 18:22:21][INFO] Initializing logger 2020-02-26 18:22:21.123407
[2020/02/26 18:22:21][INFO] ----------------------------------------------
[2020/02/26 18:22:21][INFO] $ C:\h\w\A4080960\w\B66A0983\u\Startup\Startup.exe --app-exe C:\h\w\A4080960\p\Core_Root\crossgen.exe --metric-type ProcessTime --trace-file-name "Crossgen Throughput - Microsoft.CodeAnalysis.CSharp.dll_startup.etl" --process-will-exit true --iterations 1 --timeout 50 --warmup true --gui-app false --working-dir C:\h\w\A4080960\p\Core_Root --report-json-path traces\perf-lab-report.json --trace-directory traces --scenario-name "Crossgen Throughput - Microsoft.CodeAnalysis.CSharp.dll" --app-args "/nologo /p C:\h\w\A4080960\p\Core_Root C:\h\w\A4080960\p\Core_Root\Microsoft.CodeAnalysis.CSharp.dll"
[2020/02/26 18:22:24][INFO] Running C:\h\w\A4080960\p\Core_Root\crossgen.exe (args: "/nologo /p C:\h\w\A4080960\p\Core_Root C:\h\w\A4080960\p\Core_Root\Microsoft.CodeAnalysis.CSharp.dll")
[2020/02/26 18:22:24][INFO] Working Directory: C:\h\w\A4080960\p\Core_Root
[2020/02/26 18:22:24][INFO] Iteration set up:  (args: )
[2020/02/26 18:22:24][INFO] Iteration clean up:  (args: )
[2020/02/26 18:22:24][INFO] =============== Warm up ================
[2020/02/26 18:22:24][INFO] ***Test***
[2020/02/26 18:22:30][INFO] Native image C:\h\w\A4080960\p\Core_Root\Microsoft.CodeAnalysis.CSharp.ni.dll generated successfully.
[2020/02/26 18:22:30][INFO] 
[2020/02/26 18:22:30][INFO] 
[2020/02/26 18:22:30][INFO] 
[2020/02/26 18:22:30][INFO] 
[2020/02/26 18:22:30][INFO] =============== Iteration 0 ================
[2020/02/26 18:22:30][INFO] ***Test***
[2020/02/26 18:22:34][INFO] Native image C:\h\w\A4080960\p\Core_Root\Microsoft.CodeAnalysis.CSharp.ni.dll generated successfully.
[2020/02/26 18:22:34][INFO] 
[2020/02/26 18:22:34][INFO] 
[2020/02/26 18:22:34][INFO] 
[2020/02/26 18:22:34][INFO] 
[2020/02/26 18:22:34][INFO] Parsing..
[2020/02/26 18:22:34][INFO] Metric         |Average        |Min            |Max
[2020/02/26 18:22:34][INFO] ---------------|---------------|---------------|---------------
[2020/02/26 18:22:34][INFO] Process Time   |3905.544 ms    |3905.544 ms    |3905.544 ms
[2020/02/26 18:22:34][INFO] Time on Thread |3680.664 ms    |3680.664 ms    |3680.664 ms
[2020/02/26 18:22:34][INFO] =============== Profile Iteration ================
[2020/02/26 18:22:35][INFO] ***Test***
[2020/02/26 18:22:39][INFO] Native image C:\h\w\A4080960\p\Core_Root\Microsoft.CodeAnalysis.CSharp.ni.dll generated successfully.
[2020/02/26 18:22:39][INFO] 
[2020/02/26 18:22:39][INFO] 
[2020/02/26 18:22:39][INFO] 
[2020/02/26 18:22:39][INFO] 
[2020/02/26 18:22:40][INFO] Merging profile..
```
(It only ran one iteration beacuse it wasn't set to upload to the lab. It will run 5 normally.)

**Comment by BruceForstall** [comment]
cc @briansull

**Comment by billwert** [comment]
Merging as the perf legs are passed and that's the only thing impacted by this change.

**Pull request merged**